pkg/kubelet/config/file_linux_test.go

```diff
@@ -250,7 +223,18 @@ func (tc *testCase) writeToFile(dir, name string, t *testing.T) string {
 	return fileName
 }
 
-func watchFileAdded(watchDir bool, t *testing.T) {
+func (tc *testCase) writeSymblicLink(dir, linkedDir, name string, t *testing.T) string {
```

Comment:
Why is this a method on testCase? It doesn't require any of the state in the testCase object.